Dockerfile

```diff
@@ -1,5 +1,27 @@
-FROM alpine
-COPY bin/helpwave /usr/local/bin/helpwave
-RUN chmod +x /usr/local/bin/helpwave
-EXPOSE 3000
-CMD /usr/local/bin/helpwave
+FROM golang:1.19 AS builder
+
+WORKDIR /app
+COPY . /app
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o helpwave .
+
+
+FROM alpine AS production
+
+LABEL maintainer="development@helpwave.de"
+
+WORKDIR /app
+
+COPY --from=builder /app/helpwave .
+RUN chmod +x helpwave
+
+RUN touch .env
```

Comment:
Is this needed? If so we should probably fix that

Reply:
Yes, it is. We should not depend on `.env` files at all.

Reply:
Created an issue for that: #52